nautobot/dcim/models/__init__.py

```diff
@@ -27,7 +27,6 @@
 from .locations import Location, LocationType
 from .power import PowerFeed, PowerPanel
 from .racks import Rack, RackGroup, RackReservation
-from .sites import Region, Site
```

Comment:
What's our plan for testing and documenting the app migration strategy for apps that were using these models? IMHO that needs to be solid before this can be merged.

Reply:
#3350